homeassistant/components/sensor/linky.py

```diff
@@ -35,26 +35,18 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     password = config[CONF_PASSWORD]
     timeout = config[CONF_TIMEOUT]
 
-    from pylinky.client import LinkyClient, PyLinkyError
-    client = LinkyClient(username, password, None, timeout)
-
-    try:
-        client.fetch_data()
-    except PyLinkyError as exp:
-        _LOGGER.error(exp)
-        return
-
-    devices = [LinkySensor('Linky', client)]
+    devices = [LinkySensor('Linky', username, password)]
```

Comment:
This [was already discussed](https://github.com/home-assistant/home-assistant/pull/16468#pullrequestreview-157116725) in the PR to include the platform.

Reply:
Fix in commit https://github.com/home-assistant/home-assistant/pull/17110/commits/ce363ee73af2d54b0dab40e7a9150e00b1404518.